doc/source/data/dataset.rst

```diff
@@ -237,6 +237,9 @@ Supported Input Formats
    * - 🤗 (Hugging Face) Dataset
      - :func:`ray.data.from_huggingface()`
      - ✅
+   * - MongoDB
+     - :func:`ray.data.read_mongo()`
+     - ✅
```

Comment:
We also need add `ray.data.write_mongo()` at line 272 below.

Reply:
Done.